pkg/tables/tables.go

```diff
@@ -56,7 +56,8 @@ var (
 
 	// Used for acorn image related printing
 	ImageAcorn = [][]string{
-		{"Repository", "{{if eq .Repository \"\"}}<none>{{else}}{{.Repository}}{{end}}"},
+		// 		{"Repo", "{{if and (not .Repo) (not .Project) }}<none>{{ else if and (not .Project) .Repo }}{{ .Repo }}{{ else if and (not .Repo) .Project }}{{ .Project }}{{ else }}{{ .Project }}::{{ .Repo }}{{ end }}"},
+		{"Repo", "{{if (not .Repo)}}<none>{{else if .Project }}{{.Project}}::{{end}}{{ .Repo }}"},
```

Comment:
Could have moved this logic into the images.go cli code, but figured it made more sense to keep it logically separated and compile together here.